Tella/Components/Connections/ReportsPageView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ReportsPageView.swift
+//  Tella
+//
+//  Created by Gustavo on 31/07/2023.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct ReportsPageView: View {
+    @Binding var selectedCell : Pages
+    @Binding var pageViewItems : [PageViewItem]
+    @Binding var draftReports : [Report]
+    @Binding var outboxedReports : [Report]
+    @Binding var submittedReports : [Report]
+    var navigateToAction : () -> Void
+    
+    var body: some View {
+                    
+            VStack(alignment: .center) {
+                        
+                PageView(selectedOption: $selectedCell, pageViewItems: $pageViewItems)
+                    .frame(maxWidth: .infinity, maxHeight: 40, alignment: .leading)
+                        
+                VStack (spacing: 0) {
+                    Spacer()
+                                
+                    switch selectedCell {
+                    
+                    case .templates:
```

Comment:
In Report view we don't have templates, I think it's better to create a separate enum for Reports and one for Uwazi